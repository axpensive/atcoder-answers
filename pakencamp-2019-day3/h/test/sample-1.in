7 3
.Q.....
....X..
.......
.X....Q
.......
Q.....X
.......