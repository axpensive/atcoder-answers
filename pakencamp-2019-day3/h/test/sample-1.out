.Q.....
.#..X..
...#.#.
.X..#.Q
......#
Q..#..X
.......